apis/supernova/src/creator.js

```diff
@@ -109,6 +109,11 @@ function createWithHooks(generator, opts, galaxy) {
           // changed is set further down only if the name is different
           this.context.theme = r.context.theme;
         }
+
+        if (r.context?.navigation) {
+          this.context.navigation = r.context.navigation;
+        }
+
```

Comment:
Not sure but have some worry that this in some edge cases) will override local (object level) navigation with global navigation

Reply:
I don't think this is needed on Render, it could be set when the object is created instead. Navigation is a global API that can't change during rendering.
So I think just moving this up to where for example this is done:
```
  const c = {
    context: {
      // static values that are not expected to
      // change during the component's life
      // --------------------
      model: opts.model,
      app: opts.app,
      global: qGlobal,
      selections: opts.selections,
      nebbie: opts.nebbie,
      element: undefined, // set on mount
      emitter: opts.emitter,
```

Reply:
I moved navigation to cell.js instead where has access of context and can set navigation